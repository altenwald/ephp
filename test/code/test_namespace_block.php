<?php

namespace Coding\Specially\MyCode {
    function hello($world) { print "hello $world\n"; }
}

?>

