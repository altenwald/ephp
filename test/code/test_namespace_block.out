
Fatal error: No code may exist outside of namespace {} in {{CWD}}/test/code/test_namespace_block.php on line 8
